Traceback (most recent call last):
  File "functionNoResult.py", line 4, in <module>
    foo(1)
  File "functionNoResult.py", line 2, in foo
    return "x"

WyppTypeError: "x"

Kein RÃ¼ckgabewert erwartet bei Aufruf der Funktion `foo`.
Aber der Aufruf gibt einen Wert vom Typ `str` zurÃ¼ck.

## Datei functionNoResult.py
## RÃ¼ckgabetyp deklariert in Zeile 1:

def foo(i: int) -> [0;31m[1mNone[0;0m:

## Fehlerhaftes return in Zeile 2:

    return "x"

## Aufruf in Zeile 4 verursacht das fehlerhafte return:

foo(1)
